File: test/fixtures/clojure/get_with_data2.clj
(require '[clj-http.client :as client])

(client/put "http://localhost:28139/v2/alerts_policy_channels.json" {:query-params {:policy_id "policy_id"
                                                                                    :channel_ids "channel_id"}
                                                                     :headers {"X-Api-Key" "{admin_api_key}",
                                                                               "Content-Type" "application/json"}})

File: test/fixtures/r/get_with_data2.r
library(httr)

headers = c(
  `X-Api-Key` = "{admin_api_key}",
  `Content-Type` = "application/json"
)

params = list(
  policy_id = "policy_id",
  channel_ids = "channel_id"
)

res <- httr::PUT(url = "http://localhost:28139/v2/alerts_policy_channels.json", httr::add_headers(.headers=headers), query = params)
